docs/android-core/participants/events.mdx

```diff
@@ -22,197 +22,283 @@ Here are the supported methods:
-            // participant stopped presenting screen in the meeting
+  <TabItem value="java" label="Java" default>
+
+```java
+    DyteParticipantsEventListener listener = new DyteParticipantEventsListener() {
+        @Override
+        public void onScreenShareUpdate(@NonNull DyteRemoteParticipant participant, boolean isEnabled) {
+            // your code here to handle screenshares from meeting
+            // you can use `meeting.participants.screenshares` to get latest screenshare participants
         }
-    }
+    };
 ```
 
+  </TabItem>
+</Tabs>
+
 ### Active speaker
 
 Triggers an event when there is any change in active speaker in the meeting.
 
+<Tabs groupId="android-core-participants">
+  <TabItem value="kotlin" label="Kotlin" default>
+
 ```kotlin
     private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onActiveSpeakerChanged(participant: DyteMeetingParticipant) {
-            // your code here to handle active speaker
-        }
-
-        override fun onNoActiveSpeaker() {
-            // your code here to handle no active speaker
+        override fun onActiveSpeakerChanged(participant: DyteRemoteParticipant?) {
+            // If participant is null, there is no active speaker
         }
     }
 ```
 
-### Pinned participant
-
-Triggers an event when there is any change in pinned participant in the meeting.
+  </TabItem>
 
-```kotlin
-    private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onParticipantPinned(participant: DyteMeetingParticipant) {
-            // your code here to show pinned participant
-        }
+  <TabItem value="java" label="Java" default>
 
-        override fun onParticipantUnpinned(participant: DyteMeetingParticipant) {
-            // your code here to remove pinned participant
+```java
+    DyteParticipantsEventListener listener = new DyteParticipantEventsListener() {
+        @Override
+        public void onActiveSpeakerChanged(@Nullable DyteRemoteParticipant participant) {
+            // If participant is null, there is no active speaker
         }
-    }
+    };
 ```
 
-### Active participants list change
+  </TabItem>
+</Tabs>
 
-Triggers an event when there is any change in active participants list in the meeting.
+### Pinned participant
+
+Triggers an event when there is any change in pinned participant in the meeting.
+
+<Tabs groupId="android-core-participants">
+  <TabItem value="kotlin" label="Kotlin" default>
 
 ```kotlin
     private val participantEventsListener = object : DyteParticipantEventsListener {
-        override fun onActiveParticipantsChanged(active: List<DyteMeetingParticipant>) {
-            // your code here to refresh active participants
+        override fun onParticipantPinned(participant: DyteRemoteParticipant) {
+        }
+
+        override fun onParticipantUnpinned(participant: DyteRemoteParticipant) {
         }
     }
 ```
 
-## Single Participant Events
```

Comment:
(change-request): "Single Participant Events" section was related to the `DyteParticipantUpdateListener`. Can we add it back and update it to V2 APIs?